code/modules/ghostroles/spawner/human/admin.dm

```diff
@@ -92,6 +92,38 @@
 	mob_name_prefix = "Ofc. "
 	mob_name_pick_message = "Pick a callsign or last-name."
 
+/datum/ghostspawner/human/admin/cciaagent/scc
```

Comment:
Keep the outfits out of standard access until the complete switchover from CCIA to SCC happens.
For now CCIA is still expected to handle the standard cases.
During non-standard cases admins can set the outfit as needed.

That said, if it really needs to be added, these paths are a abomination, turn them into something more readable akain to ccia is currently setup.
```
/datum/ghostspawner/human/admin/cciaagent/scc/bodyguard
```
Possibly turn both into subtypes of a parent type that cant be spawned in. ie.
```
/datum/ghostspawner/human/admin/corpagent/ccia
/datum/ghostspawner/human/admin/corpagent/scc
/datum/ghostspawner/human/admin/corpbodyguard/ccia
/datum/ghostspawner/human/admin/corpbodyguard/scc
```

Reply:
yeah, the team's planned to be able to spawn as SCC agents as soon as the SCC stuff hits the station, i believe. i organized the typepaths into its own folder, should look a lot better now

Reply:
changed to admin edit